ui/src/app/base/components/ContextualMenu/ContextualMenu.js

```diff
@@ -76,6 +76,18 @@ const ContextualMenu = ({
     // redefined on a state update which causes an infinite loop here.
   }, [isOpen]); // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
```

Comment:
This could possibly use a comment explaining why it's necessary.

Reply:
Done, thanks.